src/In.ProjectEKA.HipService/wwwroot/swagger.yaml

```diff
@@ -221,6 +222,69 @@ paths:
             application/xml:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
+
+  /health-information:
```

Comment:
Can we have it same as CM `/health-information/request`

Reply:
sure